modules/__init__.py

```diff
@@ -16,15 +16,15 @@
     attributeList=dir(moduleImport)
     for attrName in attributeList:
         testAttr=getattr(moduleImport,attrName)
-        if hasattr(testAttr,'__mro__'):
-            if Module in testAttr.__mro__:
-                if testAttr.vanilla:
-                    VANILLA_MODULES[moduleName]=testAttr
-                else:
-                    MODDED_MODULES[moduleName]=testAttr
-                break
+        if isinstance(testAttr,Module):
+            if testAttr.vanilla:
+                VANILLA_MODULES[moduleName]=testAttr
+            else:
+                MODDED_MODULES[moduleName]=testAttr
+            break
     else:
-        raise Exception(f"Module was not found in the script `{moduleName}.py`")
+        continue
+    raise Exception(f"Module was not found in the script `{moduleName}.py`")
```

Comment:
Uh, what? I can't see what the reasoning was for this change.

Reply:
Let me test it and see if it works and I'll get back to you